study-builder/studies/rarex/study.conf

```diff
@@ -576,34 +606,79 @@
                 },
                 {
                   "type": "TEXT",
-                  "field": "last_name",
+                  "field": "unable_first_name_1",
                   "activityCode": "CONSENT_ASSENT",
-                  "questionStableId": "PARENTAL_ASSENT_PARENT_LASTNAME"
+                  "questionStableId": "PARENTAL_ASSENT_CHILD_FIRSTNAME"
                 },
                 {
                   "type": "TEXT",
-                  "field": "first_name",
+                  "field": "unable_last_name_1",
                   "activityCode": "CONSENT_ASSENT",
-                  "questionStableId": "PARENTAL_ASSENT_PARENT_FIRSTNAME"
+                  "questionStableId": "PARENTAL_ASSENT_CHILD_LASTNAME"
                 },
                 {
                   "type": "TEXT",
-                  "field": "child_last_name",
+                  "field": "unable_signature_1",
                   "activityCode": "CONSENT_ASSENT",
-                  "questionStableId": "PARENTAL_ASSENT_CHILD_LASTNAME"
+                  "questionStableId": "CONSENT_ASSENT_PARENT_SIGNATURE"
+                },
+                {
+                  "type": "ACTIVITY_DATE",
+                  "field": "date",
+                  "activityCode": "CONSENT_ASSENT"
                 },
                 {
                   "type": "TEXT",
-                  "field": "child_first_name",
+                  "field": "person_first_name_1",
                   "activityCode": "CONSENT_ASSENT",
-                  "questionStableId": "PARENTAL_ASSENT_CHILD_FIRSTNAME"
+                  "questionStableId": "PARENTAL_ASSENT_PARENT_FIRSTNAME"
                 },
                 {
                   "type": "TEXT",
-                  "field": "signature",
+                  "field": "person_last_name_1",
+                  "activityCode": "CONSENT_ASSENT",
+                  "questionStableId": "PARENTAL_ASSENT_PARENT_LASTNAME"
+                },
+                {
+                  "type": "TEXT",
+                  "field": "person_signature_1",
                   "activityCode": "CONSENT_ASSENT",
                   "questionStableId": "CONSENT_ASSENT_PARENT_SIGNATURE"
                 },
+                {
+                  "type": "ACTIVITY_DATE",
+                  "field": "date",
```

Comment:
Do we need different PDF field names for the different "Date of Signature" fields?

Reply:
That's right, on my way to correct them